app/config/question-bank.js

```diff
@@ -1827,7 +1837,7 @@ const questionBank = {
           baseUrl: 'business-details',
           backUrl: 'score',
           nextUrl: 'applying',
-          preValidationKeys: ['current-score'],
+          preValidationKeys: ['score'],
```

Comment:
not sure if score page has a yarKey as it doesn't have any input, so using current-score works well. This can be changed back :)